lib/support/cli.js

```diff
@@ -254,6 +255,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
     }
     // End hack
 
+    // FIXME: Find a solution that would work for all aliasing
```

Comment:
As a matter of fact... I have an idea of how to do it. But we can leave this in for now.